doc/release-notes/5.9-release-notes.md

```diff
@@ -0,0 +1,168 @@
+
+### Support for HTTP "Range" Header for Partial File Downloads
+
+Dataverse now supports the HTTP "Range" header, which allows users to download parts of a file. Here are some examples:
+
+- `bytes=0-9` gets the first 10 bytes.
+- `bytes=10-19` gets 10 bytes from the middle.
+- `bytes=-10` gets the last 10 bytes.
+- `bytes=9-` gets all bytes except the first 10.
+
+Only a single range is supported. For more information, see the [Data Access API](https://guides.dataverse.org/en/5.9/api/dataaccess.html) section of the API Guide.
+
+### Support for Optional External Metadata Validation Scripts
+
+The Dataverse software now allows an installation administrator to provide custom scripts for additional metadata validation when datasets are being published and/or when Dataverse collections are being published or modified. The Harvard Dataverse Repository has been using this mechanism to combat content that violates our Terms of Use, specifically spam content. All the validation or verification logic is defined in these external scripts, thus making it possible for an installation to add checks custom-tailored to their needs.  
+
+Please note that only the metadata are subject to these validation checks. This does not check the content of any uploaded files.
+
+For more information, see the [Database Settings](https://guides.dataverse.org/en/5.9/installation/config.html) section of the Guide. The new settings are listed below, in the "New JVM Options and DB Settings" section of these release notes.
+
+### Displaying Author's Identifier as Link
+
+In the dataset page's metadata tab the author's identifier is now displayed as a clickable link, which points to the profile page in the external service (ORCID, VIAF etc.) in cases where the identifier scheme provides a resolvable landing page. If the identifier does not match the expected scheme, a link is not shown.
+
+### Auxiliary File API Enhancements
+
+This release includes updates to the Auxiliary File API. These updates include:
+
+- Auxiliary files can now also be associated with non-tabular files
+- Auxiliary files can now be deleted
+- Duplicate Auxiliary files can no longer be created
+- A new API has been added to list Auxiliary files by their origin
+- Some auxiliary were being saved with the wrong content type (MIME type) but now the user can supply the content type on upload, overriding the type that would otherwise be assigned
+- Improved error reporting
+- A bugfix involving checksums for Auxiliary files
+
+Please note that the Auxiliary files feature is experimental and is designed to support integration with tools from the [OpenDP Project](https://opendp.org). If the API endpoints are not needed they can be blocked.
+
+## Major Use Cases and Infrastructure Enhancements
```

Comment:
I don't want to  make or request changes at the very last moment that are essentially about changing our process/the established structure of these notes. But this just feels wrong, to have long, verbose descriptions of some new features, under "Release Highlits"; but then follow up with brief, concise mentions of the same features in the "Major use cases" section... 
It should really be the other way around; a brief, skinny list first; then the verbose section - ? 
Maybe even: 1) skinny section; 2) upgrade instruction; and only then, "if you're still reading" - the detailed descriptions for those few who have the attention span for such things - ? 
(once again, I don't want to change this note, for this release)